docs/dom-testing-library/api-helpers.md

```diff
@@ -148,6 +148,30 @@ cy.get('form').within(() => {
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+## `getRoles`
+
+This function allows iteration over the implicit aria roles represented in a
```

Comment:
It'd be great if we could provide a link to an exhaustive list of implicit ARIA roles. Couldn't find anything better than [this](https://www.w3.org/TR/html-aria/#document-conformance-requirements-for-use-of-aria-attributes-in-html), though.

Thoughts?

Reply:
I couldn't find much either.  That looks pretty decent though, just wish it was a bit more terse.  

Added a link to it here: [b026d9f](https://github.com/testing-library/testing-library-docs/pull/155/commits/b026d9fd87aaff435d6918a5ff1462cff28afd6e)